src/manifest.json

```diff
@@ -44,6 +44,16 @@
   "content_scripts": [
     {
       "js": [
+        "js/first_parties.js"
+      ],
+      "matches": [
+        "<all_urls>"
```

Comment:
Chrome injected scripts can access page DOM fine (they do have a [separate JS environment](https://developer.chrome.com/extensions/content_scripts#execution-environment), but we don't need that for our purposes here). So we don't need to make use of the "injected script injecting a page script" pattern.